californium-core/pom.xml

```diff
@@ -37,6 +37,35 @@
 	<build>
 		<plugins>
 			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Export-Package>
+							org.eclipse.californium.core,
```

Comment:
yes with this approach actually the same meta-data is generated as with using the individual artifacts. it just means that if the californium project decides to update the dependencies, the boundaries need to updated accordingly, which would be done automatically with the individual bundles